tests/test_enterprise/views/test_grant_data_sharing_permissions.py

```diff
@@ -606,28 +627,36 @@ def test_get_program_consent_missing_parameter(self, missing_parameter):
         valid_get_params = self.valid_get_params.copy()
         valid_get_params.pop(missing_parameter)
         self._login()
-        response = self.client.get(self.url, valid_get_params)
-        assert response.status_code == 404
+        with mock.patch('enterprise.views.render') as mock_render:
```

Comment:
looks like we have this pattern repeating in many tests: mock render and its return value, call some url, assert that the response code is 404. can this be consolidated in one method that each test that needs it calls?

Reply:
yeah can be done. in looking at this, i found a few copy paste errors (not just my own!) that needed to be fixed in this test file, so pushing that up with shortly.